models/book.js

```diff
@@ -11,16 +11,30 @@ const BookSchema = mongoose.Schema({
         required : true ,
         trim : true
     },
-    Author :{
+    Language:{
         type : String ,
+        required:true ,
+        trim : true
+    },
+    yearOfEdition :{
+        type : Number ,
         required : true ,
         trim : true
     },
     owner:{
         type : mongoose.Schema.Types.ObjectId,
         required:true,
         ref : 'Publisher'
+    },
+    owner:{
```

Comment:
@Raghad-Hodaly Why do you have 2 owner fields?
Also you should have a file to hold the location of the book (the PDF URL)